homeassistant/core.py

```diff
@@ -1972,8 +1973,26 @@ async def async_load(self) -> None:
             CORE_STORAGE_KEY,
             private=True,
             atomic_writes=True,
+            minor_version=CORE_STORAGE_MINOR_VERSION,
         )
 
+        async def _async_migrate_store(
+            old_major_version: int,
+            old_minor_version: int,
+            old_data: dict[str, Any],
+        ) -> dict[str, Any]:
+            """Migrate to the new version."""
+            data = old_data
+            if old_major_version == 1 and old_minor_version < 2:
+                # Version 1.1 moves unit_system to unit_system_key
+                data["unit_system_key"] = data.get("unit_system")
```

Comment:
When doing this, shouldn't we also translate "imperial" to "us_customary"?

Reply:
done